src/test/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtilTest.java

```diff
@@ -407,5 +417,88 @@ public void test_updateUserDetailsForActiveOrganisation() {
+        Map<String, Organisation> response = RefDataUtil.updateUserDetailsForActiveOrganisation(realResponseEntity, activeOrganisationDtls);
+
+
+        SuperUser item = ((SuperUser)users.get(0));
+        assertNull(item.getId());
+        assertEquals("fName", item.getFirstName());
+        assertEquals("lName", item.getLastName());
+        assertEquals("some@email.com", item.getEmailAddress());
+        assertNull(item.getOrganisation().getId());
+        assertEquals("Org-Name", item.getOrganisation().getName());
+    }
+
+    @Test(expected = ExternalApiException.class)
+    public void testGetSingleUserIdFromUserProfile() throws Exception {
+        Map<String, Collection<String>> header = new HashMap<>();
+        Collection<String> list = new ArrayList<>();
+        header.put("content-encoding", list);
+        String body = "{"
+            + "  \"errorMessage\": \"400\","
+            + "  \"errorDescription\": \"BAD REQUEST\","
+            + "  \"timeStamp\": \"23:10\""
+            + "}";
+
+        Response response = Response.builder().status(400).reason("BAD REQUEST").headers(header).body(body, UTF_8).request(mock(Request.class)).build();
+        when(userProfileFeignClient.getUserProfileById(any())).thenReturn(response);
+
+        ProfessionalUser result = RefDataUtil.getSingleUserIdFromUserProfile(new ProfessionalUser("firstName", "lastName", "emailAddress", new Organisation("name", OrganisationStatus.PENDING, "sraId", "companyNumber", Boolean.TRUE, "companyUrl")), userProfileFeignClient, Boolean.TRUE);
+        assertEquals(new ProfessionalUser("firstName", "lastName", "emailAddress", new Organisation("name", OrganisationStatus.PENDING, "sraId", "companyNumber", Boolean.TRUE, "companyUrl")), result);
+        verify(userProfileFeignClient, times(1)).getUserProfileById(any());
+    }
+
+    @Test
+    public void test_mapUserInfo_without_rolesTrue() {
+
+        HttpHeaders header = new HttpHeaders();
+        header.setContentType(APPLICATION_JSON);
+        ResponseEntity<?> realResponseEntity = new ResponseEntity<>(getUserProfileResponse, header, HttpStatus.OK);
+        ProfessionalUser responseUser = RefDataUtil.mapUserInfo(new ProfessionalUser(), realResponseEntity, true);
+        assertNull(responseUser.getId());
+        assertEquals("firstName", responseUser.getFirstName());
+        assertEquals("lastName", responseUser.getLastName());
+        assertEquals("email@org.com", responseUser.getEmailAddress());
+        assertNull(responseUser.getOrganisation());
+
+        assertNull(responseUser.getDeleted());
+        assertNull(responseUser.getLastUpdated());
+        assertNull(responseUser.getCreated());
+        assertEquals("pui-case-manager", responseUser.getRoles().get(0));
+
+        assertEquals(uk.gov.hmcts.reform.professionalapi.controller.constants.IdamStatus.ACTIVE, responseUser.getIdamStatus());
```

Comment:
import the package  and use only IdamStatus.Active here

Reply:
Done